### ISSUE TITLE
Add cli command to train a label finetune model

### ISSUE THREAD

**Issue by nadahlberg** [freelawproject/classifier-experiments#117]
Add a CLI with default settings for training a binary classification model for a given label. Should pull training data via load_trainset and run a boilerplate training run.
